package.json

```diff
@@ -6,51 +6,51 @@
   "license": "Apache-2.0",
   "homepage": "https://github.com/softonic/axios-retry",
   "files": [
-    "es",
-    "lib",
-    "index.js",
     "index.d.ts"
   ],
```

Comment:
`index.d.ts` was removed so should be removed here.
And while `dist` might be included by default, it wouldn't hurt to add it explicitly. (Though that's personal preference at this point)